fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/visitors/MetadataPlanVisitor.java

```diff
@@ -78,4 +79,24 @@ public QueryPlan.MetadataQueryPlan visitSimpleDescribeSchemaTemplateStatement(@N
         final var schemaTemplateId = visitUid(ctx.uid());
         return QueryPlan.MetadataQueryPlan.of(ddlFactory.getDescribeSchemaTemplateQueryAction(schemaTemplateId.getName()));
     }
+
+    // TODO should copy go on a different visitor?
```

Comment:
@hatyo I put this under `MetadataPlanVisitor`, but I'm not sure if it is more appropriate to have it in a new visitor? The other visitors didn't look more correct.